frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -392,16 +393,28 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bypass_postselect = kwargs.pop("bypass_postselect", False)
```

Comment:
Is `bypass_postselect` ever passed on initialization?

Reply:
No, but it could.

Reply:
removed